learn/how-to-keep-ballerina-up-to-date.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: ballerina-inner-page
+title: How to Keep Ballerina up to date
+permalink: /learn/how-to-keep-ballerina-up-to-date/
+---
+
+# How to Keep Ballerina up to date
+This guide explains how to maintain your Ballerina installation up to date with the latest patch and minor releases. If you haven’t installed Ballerina yet, visit [Getting Started](https://ballerina.io/learn/getting-started/) guide. 
```

Comment:
Getting started has changed to "Installing Ballerina" with the PR [1] . Should we udpate the link with this?

Reply:
```suggestion
This guide explains how to maintain your Ballerina installation up to date with the latest patch and minor releases. If you haven’t installed Ballerina yet, visit [installing Ballerina](https://ballerina.io/learn/getting-started/). 
```